ls1intum/Artemis
# Calculate the current server time to be used in a better way to prevent problems

### Checklist

### Motivation and Context
We now use a custom time request which should be more reliable.
We also calculate the current time based on the offset between client and server and not based on a counter using the last synced server time (which is unreliable in background tabs)

### Description
The call is invoked at most 5 times per minute.
Note: Even if the student manipulates his system time, the artemis time won't be in sync for max. 2 minutes if online and in an exam.

### Steps for Testing
1. Student: Navigate into an exam with a wrong local computer time / different time zone
2. The time until the exam end should be displayed correctly 

Note: If you change the time locally while being in this view, this will not be discovered
